### PR TITLE
Add link to Github commit

### PR DESCRIPTION
**Pull request by shayanb**
Add a hyperlink on the commit date to open the Github commit in a new page.

### CODE REVIEW

**Comment by pomber** [comment]
Thanks, I like it. I prefer to show the link only for the active commit to avoid accidentally clicking on the link when trying to change the commit. You can use `isActive` to choose what to render.  
Also, can yo remove the extra space `{" "}` at the end of line 45?

**Comment by pomber** [comment]
One more thing, check if there is a commitUrl: `isActive && commit.commitUrl`

**Comment by shayanb** [comment]
@pomber thanks, made the changes. Sorry if my method is strange, I'm not really a React dev :)

File: src/history.js [hunk 1]
```diff
@@ -40,7 +41,9 @@ function CommitInfo({ commit, move, onClick }) {
             {commit.author.login}
           </div>
           <div style={{ fontSize: "0.85rem", opacity: "0.9" }}>
-            on {commit.date.toDateString()}
+            <a href={Activelink} target="_blank">
+              <div>{commit.date.toDateString()}</div>
+            </a>
           </div>
```
Comment (pomber) [inline src/history.js]: You can do something like this inside the div:
```jsx
  {isActive && commit.commitUrl ? (
    <a href={commit.commitUrl} target="_blank">
      on {commit.date.toDateString()}
    </a>
  ) : (
    `on ${commit.date.toDateString()}`
  )}
```

Comment (shayanb) [reply src/history.js]: Ah I was trying to do it inside the div, but using `if`. that's why it was failing me.

**Comment by pomber** [comment]
Thanks!

**Pull request merged**